bassa.yml

```diff
@@ -0,0 +1,7 @@
+database:
+        database_type: mysql
+        database_port: 3306
+        database_ip: localhost
+        database_name: bassa
+        database_user_username: bassaman
```

Comment:
shall we remove hardcoded usernames and passwords? WDYT?

Reply:
Yes we can do that. Anyways, the bassa.yml file is also prepared by another script that I wrote for one of the tasks in GCI, so I guess these can be removed from here